---
output: word_document
---
# Introduction

Why we care about deleterious variants.  
Discuss results of Mezmouk 2014.   
We are extending this in three ways  

  * all deleterious SNPs not just coding  
  * genome-wide not just reduced representation  
  * using GS to test whether they improve prediction  

# Methods

## Samples and Field Stuff

samples, diallel construction, field methods

## Phenotype analysis

heritability etc.

## Sequencing

sequence methods, alignment, snp calling
comparison of our SNPs to GBS, 55K, HM2

### IBD analysis

### Gerp analysis

## Association tests

### GWAS

### GS

# Results

## Genotypes

SNPs, accuracy, IBD analysis

## Phenotypes

 * percent heterosis, heritability  
 * main figure: boxplot of heterosis? per parent?

## GWAS

 * Haplotype GWAS results = overdominance
 * SNPs within show dominance
 * enrichment for GERP??
 * ≥1 main figure

## Cross-validation

 * Gerp additive
 * Gerp dominant
 * ≥1 main figure
 
# Discussion

 * how do results match with heritability and heterosis?
 * do we support deleterious model of Mezmouk et al.?


